### PR TITLE
[10.0][IMP] partner_contact_configuration: Improve menus

### PR DESCRIPTION
**Pull request by rousseldenis**
Better menu presentation

image

### CODE REVIEW

**Comment by sbidoul** [comment]
@rousseldenis to facilate review, can you provide a screenshot (even better, before/after screenshots)?

**Comment by rousseldenis** [comment]
@sbidoul Ok, I'll upload one as soon runbot is come back

**Comment by rafaelbn** [comment]
Hello, still needed @rousseldenis ?

**Comment by rousseldenis** [comment]
@sbidoul @rafaelbn PR description with image done

File: partner_contact_configuration/__manifest__.py [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
     'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
     'website': 'https://acsone.eu',
-    'depends': ['base',
-                'contacts'
-                ],
-    'data': ['views/partner_contact.xml'
-             ],
-    'demo': [
+    'depends': [
+        'base',
```
Comment (NL66278) [inline partner_contact_configuration/__manifest__.py:16]: No need for base

File: partner_contact_configuration/__manifest__.py [hunk 2]
```diff
@@ -7,15 +7,15 @@
     'summary': """
         Adds menu configuration access through the 'contacts' module main menu
         """,
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```
Comment (NL66278) [inline partner_contact_configuration/__manifest__.py]: No need to change this, will be done by ocabot

File: partner_contact_configuration/__manifest__.py [hunk 3]
```diff
@@ -7,15 +7,15 @@
...
     'license': 'AGPL-3',
     'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
     'website': 'https://acsone.eu',
```
Comment (NL66278) [inline partner_contact_configuration/__manifest__.py]: website should be https://github.com/OCA/partner-contact

**Review by NL66278 — CHANGES_REQUESTED** [review]
Thank you for this PR. A few small changes would be welcome however. See other comments.

File: partner_contact_configuration/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply partner_contact_configuration/__manifest__.py:16]: I'd prefer explicit dependencies

File: partner_contact_configuration/__manifest__.py (hunk 2 above)
Comment (rousseldenis) [reply partner_contact_configuration/__manifest__.py]: It was not the case on 2018 :-)

**Comment by rousseldenis** [comment]
@NL66278 I will finish this

**Comment by rousseldenis** [comment]
@NL66278  This one is ready.

File: partner_contact_configuration/__manifest__.py (hunk 1 above)
Comment (NL66278) [reply partner_contact_configuration/__manifest__.py:16]: I do not agree in this case, but not blocking.

**Review by NL66278 — APPROVED** [review]
:+1: LGTM. Travis errors are not related.

**Comment by NL66278** [comment]
Manually merging: oca-bot failed because of base_geo_location not able to reach server. No errors in this PR.

**Comment by rousseldenis** [comment]
> Manually merging: oca-bot failed because of base_geo_location not able to reach server. No errors in this PR.

Try to avoid this as a lot of automation is done through the bot as making .pot files, upgrading version and so on. As it is quite old, we can wait again a little :smiley:

**Pull request merged**